modules/ROOT/pages/showcase-apps/service-setup.adoc

```diff
@@ -1,2 +1,295 @@
 include::{partialsdir}/attributes.adoc[]
 
+= Setting up Services for Showcase Apps
```

Comment:
Maybe add a prerequisites step directing the user to the general mobile services guide (https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/configuring-dev-env.adoc)